en/controllers/components/authentication.rst

```diff
@@ -289,9 +289,13 @@ by including them in AuthComponents authenticate array::
 
     $this->Auth->config('authenticate', [
         'Openid', // app authentication object.
-        'AuthBag.Combo', // plugin authentication object.
+        'AuthBag.Openid', // plugin authentication object.
     ]);
 
+> Note that when using plugin notation there's no 'Authenticate' word when
```

Comment:
The Authenticate word is not related to plugins, the same can be said about authenticators placed in your main app.